### DISCUSSION TITLE
Product Backlog Meeting

### DISCUSSION THREAD

**Discussion by keducation** [keducation/se_team_16#9] [SE Team Project 03]
https://drive.google.com/file/d/1SXdDbxD6Jf_yb_AeWsXVPtDPOeKLptGF/view?usp=sharing
or
[Product Backlog Meeting.pdf](https://github.com/keducation/se_team_16/files/6237599/Product.Backlog.Meeting.pdf)

As of March 31, 2021

These are the things that must be done for Requirements 01
- A system of users
	- Administrator(s)
	- Customers
- Decide on the items that we would sell on this online shopping mall
- Necessary webpages for the base infrastructure:
	- Main page
	- Sign up page
	- Login page
	- Announcements and Events page
	- Item register, delete, manage page

We have come to the conclusion that by the end of this week, we would decide on the type(s) of item(s) we would sell on our website.

We would then come up with an elementary version of the overall design and theme.

Then we would actually design the webpages that are necessary (the front-end, GUI part of development)

Finally, we would do the main part development, which is the back-end coding.

**Comment by mii-p** [comment]
What about selling small games style? This is something I can work on actively and since I have a lot of interest on it, so I could do several ones. Besides, I also have previous works done. If you don't mind I would like to work on the product our website will sell.

**Comment by mii-p** [comment]
About the design, I think we should follow something simple like other games website. Something with a huge slide at the begin of the page and with a dark theme as a base. 

Example as: 
https://www.blizzard.com/en-us/
https://www.ubisoft.com/ko-kr/
https://www.origin.com/kor/en-us/store/deals/simssale?utm_campaign=TS4_PBM_KR_GAM_SRCH_GGL_UA-BRAND-E-KO&utm_medium=cpc&utm_source=google&utm_term=sims&ds_rl=1276234&gclsrc=ds&gclsrc=ds
https://store.steampowered.com/

All use dark theme, so I think it allows us to have an idea of sophistication, which I find it cool.
